crates/next-custom-transforms/tests/fixture/server-actions/client/6/output.js

```diff
@@ -0,0 +1,2 @@
+/* __next_internal_action_entry_do_not_use__ {"3128060c414d59f8552e4788b846c0d2b7f74743":"$$RSC_SERVER_CACHE_0"} */ import { createServerReference, callServer, findSourceMapURL } from "private-next-rsc-action-client-wrapper";
+export var foo = /*#__PURE__*/ createServerReference("ab21efdafbe611287bc25c0462b1e0510d13e48b", callServer, void 0, findSourceMapURL, "foo");
```

Comment:
should cache functions be receiving `callServer`? or is that just an artifact of reusing `createServerReference`

Reply:
It's (almost) the same implementation as Actions so basically reusing it 👍